plaidml/keras/backend.py

```diff
@@ -1073,26 +1073,48 @@ def ndim(x):
 @_log_call
 def normalize_batch_in_training(x, gamma, beta, reduction_axes, epsilon=1e-3):
     rank = x.shape.ndims
-    if reduction_axes == None:
-        axes = [rank - 1]
+    if rank == 4 and reduction_axes in [[0, 1, 2], [0, 2, 3]]:
+        # NOTE: Tensorflow's code is explicitly checking for reduction axes in
+        # the case where there are 4 dims in x. If the reduction axes are passed
+        # in explicitly when a layer is created, the broadcasting behavior
+        # differs.
+        target_shape = [1 if i in reduction_axes else i for i in range(4)]
```

Comment:
The `else i` needs to be replaced with something that grabs the shape at axis `i`; similarly for the corresponding code int he PlaidML2 bridge